app/src/renderer/components/common/NiPart.vue

```diff
@@ -36,16 +36,16 @@ export default {
 .ni-part-container
   margin 0 auto
 
+  &:last-child
+    padding-bottom: 1.5rem
```

Comment:
Can we get this to affect 768px and higher? The padding is too much on small screens. Looks fantastic on larger screens though.

Reply:
yep!